pages/ref/_id/index.vue

```diff
@@ -129,6 +134,22 @@ export default {
       ],
     };
   },
+  watch: {
+    getUserIsFetching(f) {
+      if (!f) {
+        if (this.getUserIsRegistered) {
+          this.$router.push({ name: 'in-tokensale' });
+        }
+      }
+    },
+  },
+  mounted() {
+    if (!this.getUserIsFetching) {
+      if (this.getUserIsRegistered) {
+        this.$router.push({ name: 'in-tokensale' });
+      }
+    }
+  },
```

Comment:
Is it able to redirect to `in-tokensale` in `asyncData` func just like register page?